default.nix

```diff
@@ -4,7 +4,14 @@
 
 with pkgs.haskell.lib;
 
-let inherit (pkgs) lib; in rec {
+rec {
+  inherit (import ./dep/gitignore.nix { inherit (pkgs) lib; }) gitignoreSource;
+  lib = pkgs.callPackage ./lib.nix {
+    inherit gitignoreSource;
+    inherit (pkgs) fetchgitPrivate;
```

Comment:
Is this needed?

Reply:
yea, idk why. i know `fetchgitPrivate` has since been removed, but it should still show up w/ callPackage on our checkout. I wanted to look further into this.